Jenkinsfile

```diff
@@ -152,6 +152,12 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             """)
         }
 
+        stage('Test QEMU Image') {
```

Comment:
I think we want this after the next block below, i.e. only if it's actually a new build.

Reply:
Ah yes, for whatever reason I thought that the new build check had already been done. Moving it below the next block.